modules/openapi-generator/src/main/resources/Java/libraries/okhttp-gson/anyof_model.mustache

```diff
@@ -210,13 +210,11 @@ public class {{classname}} extends AbstractOpenApiSchema{{#vendorExtensions.x-im
         super("anyOf", {{#isNullable}}Boolean.TRUE{{/isNullable}}{{^isNullable}}Boolean.FALSE{{/isNullable}});
     }
 
-    {{#anyOf}}
-    public {{classname}}({{{.}}} o) {
+    public {{classname}}(Object o) {
```

Comment:
instead of hardcoding to Object, what about using the `{{^vendorExtensions.x-duplicated-data-type}} ... {{/vendorExtensions.x-duplicated-data-type}}` tag instead (please see line 51 for an example)?